.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @DataDog/apm-go @DataDog/asm-go
```

Comment:
Wouldn't it be better to use @dd-trace-go-guild to also include people fron profiling ?

Reply:
They have a tiny, tiny integration and I would rather avoir imposing notification fatigue on them